### PR TITLE
fix: added border color prop type

### LINKED ISSUE #45: Missing propTypes.borderColor
The `propTypes` export doesn't define borderColor, so the cleanElement HOC doesn't strip this prop and it's passed down to the styled element.

### PR DESCRIPTION
**Pull request by theutz**
This seemed like a pretty straightforward fix
Fixes #45

### CODE REVIEW

**Comment by theutz** [comment]
I also noticed that the PropTypes listed for box shadow do not align with the behavior exhibited, nor with the documentation. So I made a lil' fix!

File: src/prop-types.js [hunk 1]
```diff
@@ -92,8 +92,15 @@ const borderWidth = {
   borderLeft: PropTypes.bool
 }
 
+const borderColor = {
```
Comment (jxnblk) [inline src/prop-types.js]: Thanks for this! This will also need to be exported from the module

Comment (theutz) [reply src/prop-types.js]: Good catch, @jxnblk. I took care of it.

**Comment by jxnblk** [comment]
LGTM! Thanks again!

**Pull request merged**